src/debugging/DockerPlatformHelper.ts

```diff
@@ -12,11 +12,10 @@ interface DockerPlatformConfiguration {
     netCore?: unknown;
     node?: unknown;
     python?: unknown;
-    preLaunchTask?: string;
 }
 
 export function getPlatform<T extends DockerPlatformConfiguration>(configuration: T): DockerPlatform | undefined {
-    if (netSdkDebugHelper.isDotNetSdkBuild(configuration?.preLaunchTask) && configuration.netCore !== undefined) {
+    if (netSdkDebugHelper.isDotNetSdkBuild(configuration)) {
```

Comment:
You don't need this check anymore; if `netSdkDebugHelper.isDotNetSdkBuild` returns true, `configuration.platform === 'netCore'` will also be true.

Reply:
If this was removed, we wouldn't be able to execute the tasks. Since we still need to use the `NetSdkDebugHelper` instead of the `NetCoreDebugHelper`.

Reply:
At the very least, you shouldn't multi-purpose isDotNetSdkBuild for both build tasks and debug tasks; the types  and checks are different and they should each have their own implementation to avoid using `unknown` in your types.

Reply:
yeah, that makes sense to me!